components/02-molecules/modal/modal.stories.js

```diff
@@ -0,0 +1,38 @@
+import modalTwig from './modal.twig';
+
+import modalData from './modal.yml';
+
+import './modal';
+
+/**
+ * Storybook Definition.
+ */
+export default {
+  title: 'Molecules/Modal',
+};
+
+export const modalButton = () => {
+  return modalTwig({ ...modalData });
+};
+
+export const modalAjaxLink = () => {
+  return modalTwig({
+    ...modalData,
+    modal__toggle:
+      '<a href="http://localhost:6006/?path=/story/molecules-modal--modal-ajax-link">Ajax modal data</a>',
+  });
+};
+
+export const modalText = () => {
+  return modalTwig({
+    ...modalData,
+    modal__toggle: '<p>Plain Ol Text</p>',
+  });
+};
+
+export const modalMedia = () => {
+  return modalTwig({
+    ...modalData,
+    modal__toggle: '<img src="https://placeimg.com/320/180/any">',
```

Comment:
I know it's just an example, but maybe an empty alt attribute is still worth throwing in here?